sycl/include/CL/sycl/libcxx_span.hpp

```diff
@@ -0,0 +1,597 @@
+// -*- C++ -*-
+//===------------------------------ span ---------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===---------------------------------------------------------------------===//
+
+#ifndef _LIBCPP_SPAN
+#define _LIBCPP_SPAN
```

Comment:
It seems the guard should be renamed as well. Otherwise it can guard away original span from libc++ user might want to include.